src/content/docs/2/recipe/notifications.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Notifications
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+In this recipe we'll take a look at sending a native notification to your user. We'll do this by using the official notification plugin.
```

Comment:
```suggestion
In this recipe we'll take a look at sending a native notification to your user. We'll do this by using [`tauri-plugin-notification`](https://github.com/tauri-apps/plugins-workspace/tree/v2/plugins/notification).
```

Where should we link to? I know there's also the read-only mirror repos but I think it's better to reference this one? Might chat back and forth on Discord on this and see what we come up with.

Reply:
I sort of hate that they're all spread out across a bunch of repos, but I know there was a long discussion around this and sure it's like that for a good reason, so don't want to beat a dead horse. 

The inconsistency between the core repo's v2 branch and the plugin workspace is something that does drive me a bit crazy (v1, v2, dev, latest, next, 1.x, release, etc.), especially when we're having to hard-code URLs in the docs that could/will change.

So if that's something the dev team would be open to doing now to make it consistent I'd really appreciate it before we write a ton of docs.